src/index.tsx

```diff
@@ -4,7 +4,6 @@ import { BrowserRouter } from 'react-router-dom'
 import App from './app/App'
 import { AuthProvider } from 'app/hooks/useAuth'
 import { store } from 'app/state/store'
-import '@ovhcloud/ods-theme-blue-jeans'
```

Comment:
Don't work for you too ?

Reply:
?
this is the old theme, it does not exists anymore on v18